chapters/projectManagement.typ

```diff
@@ -0,0 +1,208 @@
+= Project Management
+// [] Vorgehen (Prozess, Reviews, Workflows, Qualitätssicherung)
+// [x] Projektplan, 
+//    [] Vergleich ursprüngliche Planung, effektive Ausführung
+// [x] Zeiterfassung (Stunden pro Woche/Stunden pro Task-Kategorie, wie Implementation Doku, Meeting, etc.)
+// Hinweis: Keine Liniendiagramme für die Darstellung von Zeit/Arbeitsaufwand pro Woche
+
+This section describes the approach of the project as well as the project plan and time tracking.
+In @approach, it is explained how the project was approached.
+@project-plan, looks into the planning of the project, including a comparative analysis between the initial plan and its effective implementation.
+Additionally, @time-tracking, provides a summarized overview of the allocated working hours coupled with a reflective assessment of the time invested in the reflects on the time invested.
+
+== Approach <approach>
+
+Development work occurred on both Linux and Windows utilizing CLion with the Ninja build system.
+The language server features were tested using Visual Studio Code and the clangd extension.
+
+To discuss the ongoing process a weekly meeting with the advisor was held were it was discussed what work has been done and what is planned for the next week.
+Receiving a feedback of the ongoing development is quite important as missing work can be detected and adjusted just the week after.
```

Comment:
"missing work" isch chli komisch hani sGfühl
vellecht was mit "direction were going"?
```suggestion
Receiving a feedback of the ongoing development is quite important as the direction we are going in can be assessed and adjusted in just a week.
```

Reply:
hans biz andersch gschriebe anstatt "going in" hani "going to" gno